client/src/lib/slate-pointers/PointerImportNode.tsx

```diff
@@ -213,8 +231,18 @@ class PointerImportNodePresentational extends React.Component<any, any> {
       blockEditor,
       visibleExportIds,
       nodeAsJson,
+      idOfHonestAnswerCandidate,
+      idOfMaliciousAnswerCandidate,
+      isAwaitingHonestExpertDecision,
     } = this.props;
 
+    const exportPointerId = this.props.nodeAsJson.data.pointerId;
+    const isKnownAsHonestAnswerCandidate =
+      isAwaitingHonestExpertDecision &&
+      idOfHonestAnswerCandidate === exportPointerId;
+    const isKnownAsMaliciousAnswerCandidate =
+      isAwaitingHonestExpertDecision &&
+      idOfMaliciousAnswerCandidate === exportPointerId;
```

Comment:
nitpick: swap the order of 240 & 243, so it matches the logic chain order below.